packages/app-storage/src/Query.tsx

```diff
@@ -93,17 +127,43 @@ class Query extends React.PureComponent<Props, State> {
         >
           <Component />
         </Labelled>
-        <Labelled className='storage--actionrow-button'>
-          <Button
-            icon='close'
-            isNegative
-            onClick={this.onRemove}
-          />
+        <Labelled>
+          {this.renderButtons()}
         </Labelled>
       </div>
     );
   }
 
+  private renderButtons () {
+    const { id, key } = this.props.value as StorageModuleQuery;
+
+    const buttons = [] as ReactNodeArray;
```

Comment:
Can we just use something like `Array<React.ReactNode>>` instead of pulling in `prop-types` (Thus far have not been using it at all since we rely on the TS definitions)